School.js

```diff
@@ -24,25 +26,34 @@ class School {
   /**
    * Enroll student in class
    * 
-   * @param {string} className - Name of the class
-   * @param {Student} student - Student object
-   * @return {Student} Enrolled student
+  //  * @param {string} className - Name of the class
+  //  * @param {Student} student - Student object
+  //  * @return {Student} Enrolled student
    */
-  enrollStudent(className, student) {
-    // Your code here
+  enrollStudent(className, student) { 
+    let classObj = this.classes[className].students
+    let newStudent = new Student(student.name, student.age, student.city, student.grade)
+    classObj.push(newStudent)
```

Comment:
You need to return here in order for you to be able to use the return value (`newStudent`) in your app.js file and include it in your response.